pkg/updater/resultstore/resultstore.go

```diff
@@ -279,57 +281,71 @@ func includeStatus(tg *configpb.TestGroup, sar *singleActionResult) bool {
 }
 
 // processGroup will convert grouped invocations into columns
-func processGroup(tg *configpb.TestGroup, inv *invocation) *updater.InflatedColumn {
-	if inv == nil || inv.InvocationProto == nil {
+func processGroup(tg *configpb.TestGroup, group *invocationGroup) *updater.InflatedColumn {
+	if group == nil || group.Invocations == nil {
 		return nil
 	}
 
-	started := inv.InvocationProto.GetTiming().GetStartTime()
-	groupID := inv.InvocationProto.GetId().GetInvocationId()
-	build := identifyBuild(tg, inv)
-	// override build by invocation ID
-	if build == "" {
-		build = groupID
+	col := &updater.InflatedColumn{
+		Column: &statepb.Column{
+			Name: group.GroupId,
+		},
+		Cells: map[string]updater.Cell{},
 	}
-	hint, err := started.AsTime().MarshalText()
-	if err != nil {
-		hint = []byte{}
-	}
-	col := &statepb.Column{
-		Build:   build,
-		Name:    groupID,
-		Started: timestampMilliseconds(started),
-		Hint:    string(hint),
-	}
-	cells := make(map[string]updater.Cell)
 
-	for _, targetResults := range inv.TargetResults {
-		for _, satr := range targetResults {
-			status, ok := convertStatus[satr.TargetProto.GetStatusAttributes().GetStatus()]
-			if !ok {
-				status = statuspb.TestStatus_UNKNOWN
+	groupedCells := make(map[string][]updater.Cell)
+
+	var hint int64
+
+	// extract info from underlying invocations and target results
+	for _, invocation := range group.Invocations {
+
+		if build := identifyBuild(tg, invocation); build != "" {
+			col.Column.Build = build
```

Comment:
What happens if invocations produce different build IDs, but have a consistent group ID?

Reply:
IIUC, build maintains the latest build value of the group (at least in the sponge code)

Reply:
Discussed in a separate channel :)